tests/unit/cluster/failover2.tcl

```diff
@@ -0,0 +1,62 @@
+# Check the basic monitoring and failover capabilities.
+
+start_cluster 3 6 {tags {external:skip cluster} overrides {cluster-ping-interval 1000 cluster-node-timeout 15000}} {
```

Comment:
```suggestion
start_cluster 3 4 {tags {external:skip cluster} overrides {cluster-ping-interval 1000 cluster-node-timeout 5000}} {
```
I think this should still work, and will launch a bit faster. Also do we need the 15 second failover time? The current test is taking 49 seconds, I think in part because of the high failover time. I presume this was set at 15 seconds so that it would take 60 seconds to reset the failover, but I think we could validate that with just a smaller waitfor condition? if you want to keep the 15 second timeout, I'm ok with that though.

Reply:
yes, update it to 5000, i originally set it to 15000 to make the test performance easier to observe, added a new verify_no_log_message to make sure there is no failover timeout